pandas/io/tests/test_pickle.py

```diff
@@ -32,6 +33,7 @@ class TestPickle():
     """
     _multiprocess_can_split_ = True
 
+    @pytest.fixture(autouse=True)
```

Comment:
@MaximilianR thanks, changed to use `setup_class` and all the tests run / pass. Made the same change in `test_packers` to remove autouse.